### PR TITLE
StatefulSet: Document workaround for known rolling update issue.

### PR DESCRIPTION
**Pull request by enisoc**
This adds a warning and a workaround for the following known issue, which affects StatefulSet in all Kubernetes versions that support it:

https://github.com/kubernetes/kubernetes/issues/67250

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/statefulset.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ provides a set of stateless replicas. Controllers such as
 * Deleting and/or scaling a StatefulSet down will *not* delete the volumes associated with the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
 * StatefulSets currently require a [Headless Service](/docs/concepts/services-networking/service/#headless-services) to be responsible for the network identity of the Pods. You are responsible for creating this Service.
 * StatefulSets do not provide any guarantees on the termination of pods when a StatefulSet is deleted. To achieve ordered and graceful termination of the pods in the StatefulSet, it is possible to scale the StatefulSet down to 0 prior to deletion.
+* When using [Rolling Updates](#rolling-updates) with the default
+  [Pod Management Policy](#pod-management-policies), it's possible to get into a
```
[the remark this answers is not in the document]
Comment (enisoc) [reply content/en/docs/concepts/workloads/controllers/statefulset.md]: Done.

**Comment by DanyC97** [comment]
@cody-clark fyi

**Pull request merged**